config/environments/development.rb

```diff
@@ -12,6 +12,8 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.